demisto_sdk/commands/find_dependencies/find_dependencies.py

```diff
@@ -218,15 +218,17 @@ def _search_packs_by_items_names(items_names: Union[str, list],
     @staticmethod
     def _search_packs_by_items_names_or_ids(items_names: Union[str, list],
                                             items_list: list,
-                                            exclude_ignored_dependencies: bool = True) -> set:
+                                            exclude_ignored_dependencies: bool = True,
+                                            incident_or_indicator: Optional[str] = None) -> set:
         """
         Searches for implemented packs of the given items.
 
         Args:
             items_names (str or list): items names to search.
             items_list (list): specific section of id set.
             exclude_ignored_dependencies (bool): Determines whether to include unsupported dependencies or not.
-
+            incident_or_indicator (str): 'indicator' if should focus on indicator fields,
+                 'incident' if should focus on incident fields and None to focus on both.
```

Comment:
Why not 'both' in the last case? 
And please improve the description here, it is not clear what 'focus' means